animal_crossing.py

```diff
@@ -0,0 +1,73 @@
+import sqlite3
+
+import config
+
+if config.bot.acnh_db is not None:
+	db = sqlite3.connect(config.bot.acnh_db)
```

Comment:
https://docs.python.org/3/library/sqlite3.html#accessing-columns-by-name-instead-of-by-index
`db.row_factory = sqlite3.Row`